docs/content/reference/install-configuration/tls/spiffe.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Traefik SPIFFE Documentation"
+description: "Learn how to configure Traefik to use SPIFFE. Read the technical documentation."
+---
+
+# SPIFFE
+
+Secure the backend connection with SPIFFE.
+{: .subtitle }
+
+[SPIFFE](https://spiffe.io/docs/latest/spiffe-about/overview/) (Secure Production Identity Framework For Everyone), 
+provides a secure identity in the form of a specially crafted X.509 certificate, 
+to every workload in an environment.
+
+Traefik is able to connect to the Workload API to obtain an x509-SVID used to secure the connection with SPIFFE enabled backends.
+
+## Configuration Example
+
+Enabling SPIFFE can be done by using a file (YAML or TOML) or CLI arguments.
+
+```yaml tab="File (YAML)"
+# Default Servers Transport
+serversTransport:
+  spiffe:
+    ids:
+    - spiffe://trust-domain/id1
+    - spiffe://trust-domain/id2
+    trustDomain: "spiffe://trust-domain"
```

Comment:
Maybe the default servers transport should be documented in a dedicated bloc in this section, and keep here only the documentation of the spiffe config option. @nmengin @sheddy-traefik wdyt?